RecoTauTag/RecoTau/plugins/DeepTauId.cc

```diff
@@ -2146,12 +2385,15 @@ class DeepTauId : public deep_tau::DeepTauBase {
       get(dnn::pfCand_chHad_pvAssociationQuality) =
           getValueLinear<int>(candFunc::getPvAssocationQuality(chH_cand), 0, 7, true);
       get(dnn::pfCand_chHad_fromPV) = getValueLinear<int>(candFunc::getFromPV(chH_cand), 0, 3, true);
-      get(dnn::pfCand_chHad_puppiWeight) = getValue(candFunc::getPuppiWeight(chH_cand));
-      get(dnn::pfCand_chHad_puppiWeightNoLep) = getValue(candFunc::getPuppiWeightNoLep(chH_cand));
+      get(dnn::pfCand_chHad_puppiWeight) = is_inner ? getValue(candFunc::getPuppiWeight(chH_cand, 0.7614090152166456))
+                                                    : getValue(candFunc::getPuppiWeight(chH_cand, 0.19749304616790136));
+      get(dnn::pfCand_chHad_puppiWeightNoLep) =
+          is_inner ? getValue(candFunc::getPuppiWeightNoLep(chH_cand, 0.7614090152166456))
+                   : getValue(candFunc::getPuppiWeightNoLep(chH_cand, 0.1974930285484762));
```

Comment:
`0.7614090152166456` is repeated and better be a named constant.
Is the other `0.19749304616790136` really intentionally different from `0.1974930285484762` (the fractional difference is just under 1E-7).

What is the origin of all these magic values?
Will they every need to be versioned in this code as well?
Perhaps making a const array with a good name and a reference in the comments with it would be better.

Reply:
this small difference indeed does not look relevant. @lwezenbe can you double-check please?

Reply:
These values were pulled from the deepTau training. This small difference is indeed not that relevant, for completeness the values that were obtained from the deeptau training were included as they were extracted, but at the 1E-7 level, there were actually some differences between trainings from different years as well so the difference described in your comment is indeed not relevant.